blocks/library/latest-posts/index.js

```diff
@@ -99,13 +99,15 @@ registerBlockType( 'core/latest-posts', {
 			const { latestPosts } = this.state;
 			const { setAttributes } = this.props;
 
-			if ( ! latestPosts.length ) {
+			const hasPosts = Array.isArray( latestPosts ) && latestPosts.length;
+			if ( ! hasPosts ) {
+				const placeholderContent = ! Array.isArray( latestPosts ) ? <Spinner /> : __( 'No posts found.' );
 				return (
 					<Placeholder
 						icon="admin-post"
 						label={ __( 'Latest Posts' ) }
 					>
-						<Spinner />
+						{ placeholderContent }
```

Comment:
This could be inlined here:

```jsx
{ condition
    ? <Spinner />
    : __( 'No posts found.' )
}
```

Reply:
Thanks for your feedback, I inlined the condition.